examples/porter-discourse/README.md

```diff
@@ -0,0 +1,62 @@
+
+## What bundle should do and look like (parameters and credentials etc)
+install:
+* cloud server (DigitalOcean, Azure, etc)
+    - need credentials 
+* cloud storage for user uploads, pictures (Amazon S3, Azure Blob Storage, etc)
+    - can set up backups 
+    - need credentials
+* email (Mailgun, SendGrid, Mailjet)
+    - can configure reply via email
+    - need credentials and email as parameter
+* domain name
+    - hostname parameter
+* ssl certificate (Let’s Encrypt free certificate) 
+    - need certificate and key
+    - have to configure NGINX and a docker container
+* docker as a service (viable as a bundle) / Virtual Machine (current standard)
+    - need credentials
+* postgres database
+    - need parameters for username, password, database name
+* configure SSO
+    - enable_sso parameter must be enabled
+    - sso_url: the offsite URL users will be sent to when attempting to log on
+    - sso_secret: a secret string used to hash SSO payloads
+* login via Google, Twitter, GitHub, Facebook
+* install plugins
+    - need plugin's git clone url 
+    - add it to app.yml
+    - rebuild the container
+* multisite configuration with docker
```

Comment:
I am not sure what this bullet refers to?

Reply:
In the readme instructions for install for Discourse, it talked about how if you want to you could configure multiple Discourse sites on the same server and the link it provided explained how to do multisite configuration with docker. I added a bullet under to try to explain it but we can also remove this if it isn't needed.

Reply:
I think let's remove that because it's pretty advanced and I predict that we are doing to encourage people to do a single site per bundle.